app/src/frontend/App.test.tsx

```diff
@@ -13,6 +13,7 @@ describe("App Component", () => {
     // Mock the electronAPI.request function
```

Comment:
Now that there are multiple:

```suggestion
    // Mock IPC methods
```